### ISSUE TITLE
Context search not working for non-static functions

### ISSUE THREAD

**Issue by ivan-mogilko** [adventuregamestudio/ags-manual#102]
Pressing F1 on a non-static function name searches solely for the function name, but since these are removed from the list of keywords, it cannot be found.

Idk if this is considered a manual or Editor's bug.

**Comment by morganwillcock** [comment]
Have you got an example of one that doesn't work? As far as I know, object members are looked up as `<object>.<member>` where the autocomplete can identify the object.

**Comment by ivan-mogilko** [comment]
Right, only some do not work. I noticed this with DrawingSurface.DrawString.

**Comment by morganwillcock** [comment]
I checked looking up `surface.Release();` with the cursor on the function name, and it correctly tried to lookup 'DrawingSurface.Release', and showed the correct page and heading.

For object, this had issues on the old manual because there are mutliple index matches for 'object', but on the new one I also found it to work. I checked looking up `object[0].SetProperty();` with the cursor on the function name, and it correctly tried to lookup 'Object.SetProperty', and showed the correct page and heading.

So I'm not sure what is going on, but perhaps you are using the old manual, or there is something in the game scripts that causes the lookup value to be incorrectly returned. You could check the latter by setting a breakpoint here:

https://github.com/adventuregamestudio/ags/blob/release-3.5.0/Editor/AGS.Editor/Components/HelpCommandsComponent.cs#L156

**Comment by ivan-mogilko** [comment]
This is very strange, this problem continues in most recent release (3.5.0.22) but is rather inconsistent. Some Object's functions work, while others don't. Perhaps there's some condition, but I was not able to find out yet.

**Comment by ericoporto** [comment]
We failed to solve this issue, it appears the exact connection between source code and the manual remains a bit mystical. A probable next task would be having a test script with comments saying what was and what wasn't successfully found in the manual.
